app/classifier/tasks/survey/editor.cjsx

```diff
@@ -190,35 +278,72 @@ module.exports = React.createClass
       </label>
     </div>
 
-  handleFiles: (forEachRow, afterFileHook, e) ->
+  handleImportTabs: (tab) ->
+    @setState editing: tab
+
+  handleApply: ->
```

Comment:
refactor. if initially copying task, may be simplest to just set @props.task to @state.task regardless of which or how many files added. also review Jim's comment regarding setting props directly.